app/models/user.rb

```
@@ -1121,20 +1121,43 @@ def should_subscribe_preview_be_regenerated?
+  def has_completed_payouts?
+    payments.completed.exists? ||
+      made_a_successful_sale_with_a_stripe_connect_or_paypal_connect_account?
+  end
+
+  # Add this to group it with other associations (usually at the top of the file)
+  has_many :sales, through: :products
+
+  # 🚀 New dashboard stat methods
+  def total_sales_count_all_time
+    sales.count
+  end
```

RuboCop (1.75.5)

[convention] Inconsistent indentation detected.

(Layout/IndentationConsistency)